doc/tutorial.txt

```diff
@@ -75,8 +75,18 @@ The use of a temporary buffer prevents the threads of unexpected wait during
 io access. The "Auto" mode is not implemented yet and fallback to "file" mode
 for the moment.
 
-* ftrace: Boolean. If enabled, rt-app logs in ftrace the main events of the use
-case. Default value is False.
+* logstats: Boolean. Report tasks generated statistics into a logfile. A logfile
+  will be generated for each task, within the logdir, reporting the main metrics
+  measured after the completion of each task activation.
+
+* ftrace: String. If not "none", rt-app logs in ftrace the events corresponding
+to the requested categories. Supported categories are:
+ - main  : events generated by the main rt-app task
+ - task  : events generated by a task
+ - run   : events generated by the run of a task
+ - loop  : events generated by each loop of a task
+ - stats : events reporting statistics on task's activation
+Default value is "none".
```

Comment:
you should keep some backward compatibility
please still mention true/false as deprecated

Reply:
and True should enable all traces